tests/alloydb_test.go

```diff
@@ -20,15 +20,31 @@ import (
 	"os"
 	"testing"
 
-	"cloud.google.com/go/alloydbconn/driver/pgxv4"
 	"github.com/GoogleCloudPlatform/alloydb-auth-proxy/internal/proxy"
+	_ "github.com/jackc/pgx/v5/stdlib"
 )
 
 var (
-	alloydbConnName = flag.String("alloydb_conn_name", os.Getenv("ALLOYDB_CONNECTION_NAME"), "AlloyDB instance connection name, in the form of 'project:region:instance'.")
-	alloydbUser     = flag.String("alloydb_user", os.Getenv("ALLOYDB_USER"), "Name of database user.")
-	alloydbPass     = flag.String("alloydb_pass", os.Getenv("ALLOYDB_PASS"), "Password for the database user; be careful when entering a password on the command line (it may go into your terminal's history).")
-	alloydbDB       = flag.String("alloydb_db", os.Getenv("ALLOYDB_DB"), "Name of the database to connect to.")
+	alloydbConnName = flag.String(
+		"alloydb_conn_name",
+		os.Getenv("ALLOYDB_CONNECTION_NAME"),
+		"AlloyDB instance connection name, in the form of 'project:region:instance'.",
```

Comment:
should this be URI form `project:region:instance`?

Reply:
Nope -- I'll fix that.